release-note-script/src/main/java/ReleaseNoteCreation.java

```diff
@@ -281,13 +281,22 @@ private String getProjectId() throws Exception {
           runSubProcessAndGetOutputAsReader(
               format(
                   "gh project list --owner %s --closed | awk '/%s/ {print}' | awk '/%s/ {print $1}'",
-                  this.owner, this.projectTitlePrefix, this.version));
+                  this.owner, this.projectTitlePrefix, getVersion()));
```

Comment:
Even for versions with suffixes, we use the project without the suffix. Therefore, we need to remove the suffixes.

For example, for the `scalardb-3.14.0-alpha.1` version,  we use the project `ScalarDB 3.14.0`.